include/spblas/vendor/rocsparse/multiply.hpp

```diff
@@ -107,7 +106,7 @@ class spmv_state_t {
                       std::function<void(rocsparse_handle)>>;
   // rocsparse_handle handle_;
   handle_manager handle_;
-  std::shared_ptr<allocator> alloc_;
+  rocsparse::hip_allocator<char> alloc_;
   long unsigned int buffer_size_;
   void* workspace_;
```

Comment:
```suggestion
  char* workspace_;
```
to avoid the casting

Reply:
Fixed.